Traceback (most recent call last):
  File "wgan_gp.py", line 72, in <module>
    MazeDataset(size=(opt.img_size, opt.img_size),
  File "/home/pierre/Git/gamegan/src/gan/dataset.py", line 18, in __init__
    self.maze_generator = aldousbroder.AldousBroderMazeGenerator(size=size)
  File "/home/pierre/Git/gamegan/src/gan/../maze_generators/aldousbroder.py", line 62, in __init__
    super(AldousBroder, self).__init__(size)
NameError: name 'AldousBroder' is not defined